### PR TITLE
closes #28 Updated all WHO metadata

### LINKED ISSUE #28: Consistency of metadata on Reference page
Minor updates highlighted in screenshot below:
- could the red boxes both show BMI consistently capitalised?
- i think all these should be shown together under "Metadata" blue box section rather than "Example Datasets" (technically they're both - but i think under "Metadata" is fine) - using following in `data.R`:

```
#' @keywords metadata
#' @family metadata
```

image

### PR DESCRIPTION
**Pull request by Pierre-Wallet**
#28 Updated documentation with metadata tag
#28 added tests for Head circumference metadata
#28 Capitalised bmi in the documentation

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by Pierre-Wallet** [comment]
@rossfarrugia , all updates done!

**Comment by Pierre-Wallet** [comment]
@rossfarrugia ,
could you please explain me what this "Check Expected — Waiting for status to be reported" means and what action is expected from my side?
Thanks!

**Comment by rossfarrugia** [comment]
> @rossfarrugia , could you please explain me what this "Check Expected — Waiting for status to be reported" means and what action is expected from my side? Thanks!

Ignore this one Pierre - I spotted same in some of mine. @zdz2101 can we get this one removed please from the CI checks?

Will try and review this PR tomorrow.

**Comment by zdz2101** [comment]
@rossfarrugia @Pierre-Wallet Yea I'll look into it but I think it might be related to the Check / (release/devel/oldrel) and supposed to go away once those resolve, but I'll check again

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
@Pierre-Wallet just couple of things spotted. other updates look fine.

File: tests/testthat/_snaps/WHO_growth_charts.md [hunk 1]
```diff
@@ -178,7 +178,47 @@
...
+       7     6  0.533   13.2 0.0980
+       8     7  0.632   13.2 0.0989
+       9     8  0.614   13.2 0.0987
+      10     9  0.596   13.3 0.0984
+      # i 1,847 more rows
+
+# WHO_growth_charts Test 12: WHO Head circumference-for-age for boys
 
     Code
       who_bmi_for_age_boys
```
[the remark this answers is not in the document]
Comment (Pierre-Wallet) [reply tests/testthat/_snaps/WHO_growth_charts.md]: Well, really!
Updates done, thanks @rossfarrugia ! The eye of the tiger! ;)

**Review by rossfarrugia — APPROVED** [review]
thanks @Pierre-Wallet
